components/backends/vllm/src/dynamo/vllm/engine_monitor.py

```diff
@@ -43,13 +43,26 @@ def __init__(self, runtime: DistributedRuntime, engine_client: AsyncLLM):
     def __del__(self):
         self._monitor_task.cancel()
 
+    def _shutdown_engine(self):
+        """
+        Shutdown the vLLM engine on crash scenarios, in order to free resources.
+        """
+        try:
+            logger.info("Shutting down vLLM engine to free GPU memory...")
+            self.engine_client.shutdown()
+            logger.info("vLLM engine shutdown completed")
```

Comment:
Can you remove the log statements? That will prevent on every single normal shutdown, and I feel like we already have too many "shutting down xyz" logs.